packages/kit-headless/src/components/autocomplete/autocomplete-listbox.tsx

```diff
@@ -12,7 +12,7 @@ import AutocompleteContextId from './autocomplete-context-id';
 
 import { KeyCode } from '../../utils/key-code.type';
 
-const preventedKeys = [
```

Comment:
I'm not sure the "list" prefix is necessary tbh
the type is "Array" which is a list

Reply:
It seems odd I agree - especially when they are not `export`-ed. Also see comment below for error that's caused by this